src/LinuxTracing/LinuxTracingIntegrationTest.cpp

```diff
@@ -565,7 +578,14 @@ void VerifyCallstackSamplesWithOuterAndInnerFunction(
   size_t matching_callstack_count = 0;
   uint64_t first_matching_callstack_timestamp_ns = std::numeric_limits<uint64_t>::max();
   uint64_t last_matching_callstack_timestamp_ns = 0;
+  absl::flat_hash_map<uint64_t, orbit_grpc_protos::Callstack> callstacks;
   for (const auto& event : events) {
+    if (event.event_case() == orbit_grpc_protos::CaptureEvent::kInternedCallstack) {
+      const orbit_grpc_protos::InternedCallstack& interned_callstack = event.interned_callstack();
+      callstacks[interned_callstack.key()] = interned_callstack.intern();
+      continue;
+    }
+
```

Comment:
The behavior of `LinuxTracing` hasn't changed and this test shouldn't change. This is relying on new events introduces by `OnCallstackSample` above, which should not happen in the first place.